### PR TITLE
Update featurestore tests

### PR DESCRIPTION
**Pull request by fredms**
# Description

# Checklist

- [ ] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] Pull request includes test coverage for the included changes.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb [hunk 1]
```diff
@@ -1172,7 +1173,11 @@
     "    version=\"1\",\n",
     "    feature_window_start_time=st,\n",
     "    feature_window_end_time=et,\n",
-    "    data_status=[\"None\", \"Incomplete\"],\n",
+    "    data_status=[\n",
+    "        DataAvailabilityStatus.NONE,\n",
+    "        DataAvailabilityStatus.INCOMPLETE,\n",
+    "        DataAvailabilityStatus.COMPLETE,\n",
```
Comment (qjxu) [inline sdk/python/featurestore_sample/notebooks/sdk_only/1. Develop a feature set and register with managed feature store.ipynb:1179]: Remove the "complete" one.

File: sdk/python/featurestore_sample/notebooks/sdk_only/4. Enable online store and run online inference.ipynb [hunk 2]
```diff
@@ -660,7 +667,11 @@
     "    version=\"1\",\n",
     "    feature_window_start_time=st,\n",
     "    feature_window_end_time=et,\n",
-    "    data_status=[\"None\"],\n",
+    "    data_status=[\n",
+    "        DataAvailabilityStatus.NONE,\n",
+    "        DataAvailabilityStatus.INCOMPLETE,\n",
+    "        DataAvailabilityStatus.COMPLETE,\n",
```
Comment (qjxu) [inline sdk/python/featurestore_sample/notebooks/sdk_only/4. Enable online store and run online inference.ipynb:673]: Remove incomplete and complete.

File: sdk/python/featurestore_sample/notebooks/sdk_only/5. Develop a feature set with custom source.ipynb [hunk 3]
```diff
@@ -105,7 +105,7 @@
     "\n",
     "# Please update the dir to ./Users/{your-alias} (or any custom directory you uploaded the samples to).\n",
```
Comment (qjxu) [inline sdk/python/featurestore_sample/notebooks/sdk_only/5. Develop a feature set with custom source.ipynb:106]: Update here with <your_user_alias>

File: sdk/python/featurestore_sample/project/fraud_model/pipelines/training_pipeline.yaml [hunk 4]
```diff
@@ -80,7 +80,7 @@ jobs:
 
   register_model_step:
     type: command
-    compute: azureml:cpu-cluster
+    compute: azureml:cpu-cluster-fs
```
Comment (qjxu) [inline sdk/python/featurestore_sample/project/fraud_model/pipelines/training_pipeline.yaml:83]: Do we need to update the inference pipeline as well?

Comment (fredms) [reply sdk/python/featurestore_sample/project/fraud_model/pipelines/training_pipeline.yaml:83]: inference pipeline was already updated to use -fs.

File: sdk/python/featurestore_sample/notebooks/sdk_only/4. Enable online store and run online inference.ipynb (hunk 2 above)
Comment (fredms) [reply sdk/python/featurestore_sample/notebooks/sdk_only/4. Enable online store and run online inference.ipynb]: removed.

File: sdk/python/featurestore_sample/notebooks/sdk_only/5. Develop a feature set with custom source.ipynb (hunk 3 above)
Comment (fredms) [reply sdk/python/featurestore_sample/notebooks/sdk_only/5. Develop a feature set with custom source.ipynb]: updated.

**Review by ynpandey — APPROVED** [review]

**Review by qjxu — APPROVED** [review]

**Review by sdgilley — APPROVED** [review]
Docs approval.

**Pull request merged**